File: Array & String/Remove Duplicates from Sorted Array II/RemoveDuplicates.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() <= 2) {
            return nums.size();
        }
        int uniqueCount = 2;
        for (int i = 2; i < nums.size(); i++) {
            if (nums[i] != nums[uniqueCount - 2]) {
                nums[uniqueCount] = nums[i];
                uniqueCount++;
            }
        }
        return uniqueCount;
    }
};

File: Array & String/Remove Duplicates from Sorted Array II/RemoveDuplicates.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j = 2;

        for (int i = 2; i < nums.length; i++) {
            if (nums[i] != nums[j - 2]) {
                nums[j++] = nums[i];
            }

        }
        return j;
    }
}